volsync.yaml

```diff
@@ -0,0 +1,72 @@
+package:
+  name: volsync
+  version: "0.13.0"
+  epoch: 0
+  description: Asynchronous data replication for Kubernetes volumes
+  copyright:
+    - license: AGPL-3.0-or-later
```

Comment:
Based on dockerfile, the package also uses:
- `rclone`
- https://github.com/dop251/diskrsync (External, last released 2022)
- https://github.com/backube/volsync/tree/main/diskrsync-tcp (Maintained)

We already have `rclone`. We need to package `diskrsync-tcp` as `volsync-diskrsync-tcp` as subpackage here.